app-modules/service-management/src/Observers/ServiceRequestObserver.php

```diff
@@ -88,5 +91,9 @@ public function saved(ServiceRequest $serviceRequest): void
         ) {
             $serviceRequest->respondent->notify(new SendEducatableServiceRequestClosedNotification($serviceRequest));
         }
+
+        if (Gate::check(Feature::FeedbackManagement->getGateName()) && $serviceRequest?->priority?->type?->has_enabled_feedback_collection && $serviceRequest?->status?->classification == SystemServiceRequestClassification::Closed && ! $serviceRequest?->feedback()->count()) {
```

Comment:
Let's break each of these onto new lines starting with the first `&&` for better readability.

Also we should add a Penant feature flag here that we check for. Something like `service-request-feedback`